doc/src/manual/arrays.md

```diff
@@ -722,9 +714,20 @@ array indirectly.  By putting the [`@views`](@ref) macro in front of an expressi
 block of code, any `array[...]` slice in that expression will be converted to
 create a `SubArray` view instead.
 
-`StridedVector` and `StridedMatrix` are convenient aliases defined to make it possible for Julia
-to call a wider range of BLAS and LAPACK functions by passing them either [`Array`](@ref) or
-`SubArray` objects, and thus saving inefficiencies from memory allocation and copying.
+A "strided" array is stored in memory with elements laid out in regular offsets such that
+an instance with a supported `isbits` element type can be passed to
+external C and Fortran functions that expect this memory layout. Strided arrays
+must define a [`strides(A)`](@ref) method that returns a tuple of "strides" for each dimension; a
+provided [`stride(A,k)`](@ref) method accesses the `k`th element within this tuple. Increasing the
+index of dimension `k` by `1` should increase the index `i` of [`getindex(A,i)`](@ref) by
+[`stride(A,k)`](@ref). If a pointer conversion method [`Base.unsafe_convert(Ptr{T}, A)`](@ref) is
```

Comment:
Is this true? Is `getindex(A,i)` a way to directly access elements based on offsets from stride calculations, or is it really still linear indexing that assumes `getindex(A,i_1,i_2,...) = getindex(A, i_1 + size(A,1)*(i_2-1) + ...)`?

Reply:
Ah, yes, you're right, thank you.  `getindex` is totally and completely decoupled from strides.